### DISCUSSION TITLE
Welcome to project-elaichi Discussions!

### DISCUSSION THREAD

**Discussion by DesignrKnight** [dscnitrourkela/project-elaichi#97] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by sudhansu3299** [comment]
So, can we start working in any of the issues mentioned? It will also be helpful if there is particular ordering(tags like urgent) of the requirements so that contributors can work on the deliverables in a hassle free manner.

**Reply by sudhansu3299** [reply]
Setting up the project board might help.
